documentation/napari-plugin/napari-plugin.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Napari Plugin
+tagline: Use some of the ilastik features in napari
+category: Documentation
+group: basic-documentation
+weight: 1
+---
+
+<div class="panel panel-warning">
+  <div class="panel-heading">
+    <h3 class="panel-title">Experimenal Software</h3>
+  </div>
+  <div class="panel-body">
+    This plugin is not stable yet.
+    Please do not expect backwards compatibility.
```

Comment:
can users save a project, currently - meaning, is there anything that could break in a backwards incompatible way?

Reply:
I put this warning mostly as a "legal disclaimer". Maybe we'll change the GUI, feature computations, or something else.